README.Mac.md

```diff
@@ -193,7 +193,7 @@ b. Create the file `ext-xdebug.ini` with this command:
 sudo touch /usr/local/etc/php/<your_php_version>/conf.d/ext-xdebug.ini
 ```
 
-c. Type the following into the `ext-xdebug.ini` file you just created (Note: replace the `zend_extension` path with the correct one on your machine):
+c. Type the following into the `ext-xdebug.ini` file you created (Note: replace the `zend_extension` path with the correct one on your machine):
```

Comment:
Again -- this is not the use of `just` we're looking to expunge...

Reply:
3 approaches in this case:

1. Reword the sentence 
```suggestion
c. Type the following into the `ext-xdebug.ini` file create the above command (Note: replace the `zend_extension` path with the correct one on your machine):
```

2. Add an exception like `<!--alex ignore just-->`, but Dave asked me not to do this.

3. Let this be flagged each time the tool is run, generating noise in the report that we have no intention of fixing.

My favourite solution is the 1st.